qa/embedded/pom.xml

```diff
@@ -1,4 +1,20 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright © 2021 camunda services GmbH (info@camunda.com)
```

Comment:
🔧 We have now 2023 :D

Reply:
I don't know how to configure the plugin to take the current hear 😄